_includes/contribution-guide-nav.html

```diff
@@ -30,20 +30,38 @@
           </div>
         </div>
   </a>
-  <a href="/documentation/contributing/releases/">
-    {% if page.url == "/documentation/contributing/releases/" %}
+  <div class="font-other-page"><b>See Pull Requests on GitHub:</b></div>
+  <a href="https://github.com/ros-planning/moveit/pulls">
```

Comment:
```suggestion
  <a href="https://github.com/ros-planning/moveit/pulls" target="_blank">
```
Lets make this a new tab since its an external link